streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/DefaultStateConstrainedBalancedAssignor.java

```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.kafka.streams.processor.internals.assignment;
 
+import static org.apache.kafka.streams.processor.internals.assignment.RankedClient.tasksToCaughtUpClients;
+
```

Comment:
req: My IDE says that this import is not needed. Since the imported method is in the same package, I agree with my IDE. Could you please remove it? Isn't strange that checkstyle did not find this?

Reply:
Good catch! It does seem weird that this slipped past checkstyle, although I've seen that happen before with an unused import I found on trunk. 🤷‍♀